.scripts/run_test.sh

```diff
@@ -22,35 +22,10 @@ cd $root_path/.test
 
 set -e
 
-if [ "$DATABASE" == "bigquery" ]; then
-
-  BIGQUERY_CREDS=$(cat ${BIGQUERY_CREDS:-$CREDENTIALS})
-
-  if [ -n "$BIGQUERY_CREDS" ]; then
-
-    # If creds provided via env var or argument, set trap to clean up, then create creds file.
-    cleanup() {
-      echo "run_test: Removing credentials file"
-      rm -f $root_path/tmp/bq_creds.json
-    }
-    trap cleanup EXIT
-
-    mkdir -p "${root_path}/tmp"
-    echo "run_test: writing bq creds to file"
-    echo $BIGQUERY_CREDS > $root_path/tmp/bq_creds.json
-
-  fi
-
-  # Set GOOGLE_APPLICATION_CREDENTIALS env var.
-  export GOOGLE_APPLICATION_CREDENTIALS=${GOOGLE_APPLICATION_CREDENTIALS:-$root_path/tmp/bq_creds.json}
-
-else
-
-  # If not BQ, take the relevant env var if it exists, set it to whatever's provided otherwise.
-  export REDSHIFT_PASSWORD=${REDSHIFT_PASSWORD:-$CREDENTIALS}
-  export SNOWFLAKE_PASSWORD=${SNOWFLAKE_PASSWORD:-$CREDENTIALS}
-
-fi
+# Take the relevant env var if it exists, set it to whatever's provided otherwise.
```

Comment:
Removing all this code is quite satisfying, very glad this change came out of the review.

I think we should have a specific issue for this in the changelog as it's a change in behaviour - for when you rebase before release @bill-warner . :)